pkg/meta/redis.go

```diff
@@ -605,6 +635,10 @@ func (m *redisMeta) totalInodesKey() string {
 	return m.prefix + totalInodes
 }
 
+func (m *redisMeta) aclKey(id uint32) string {
```

Comment:
It's better to have all these in single HashMap

Reply:
Each get acl request also required the generation of an ACL key, but after testing, the impact on CPU and memory usage was minimal, so it has been removed.

Reply:
done